src/app/themes/volumio/components/track-manager/volumio-track-manager.html

```diff
@@ -39,9 +39,21 @@
         <div class="smallText">
           {{trackManager.playerService.songLength}}
         </div>
-        <div class="footerText">
-          <span ng-if="trackManager.playerService.state.trackType !== 'webradio'">
-            {{trackManager.playerService.state.trackType}}
+        <div class="footerText" title="Audio Format">
+          <span ng-if="trackManager.playerService.state.trackType !== 'webradio'" ng-switch on="trackManager.playerService.state.trackType">
```

Comment:
Here you are adding a LOT of watchers, this is not good for performance, you should map the available audio format and assign to a variable, then use that variable as ng-src value, so you'll have only two watcher and one line of html. Also the markup is too much verbose.